### ISSUE TITLE
Algorithm accuracy

### ISSUE THREAD

**Issue by HopePersist** [YonghaoXu/SSFCN#2]

**Comment by HopePersist** [comment]
sorry to bother you again.
When I run your program, I'm not sure that the results of the program are correct.
My GPU is a Nvidia 1650, the Version of TensorFlow-gpu is 1.13.1. The training sample size is nnn = [66, 186, 21, 31, 13, 50, 13, 37, 10](1%). I haven't changed any other parameters or Settings
But the OA is about 0.94.
Because the accuracy of your article is too different, so I want to consult you whether this data is reliable.
If I want to reference your article, can I use this data

**Comment by YonghaoXu** [comment]
The experimental results in our paper are obtained with ``150`` training samples per class. Since you have used a much smaller training set, I think the drop in OA is reasonable.

**Comment by HopePersist** [comment]
Thank you for your reply.
I will read your article detailedly and cite your methods in my future work.
